pkg/datapath/linux/node.go

```diff
@@ -523,7 +532,7 @@ func (n *linuxNodeHandler) enableSubnetIPsec(v4CIDR, v6CIDR []*net.IPNet) {
 			if err != nil {
 				upsertIPsecLog(err, "getV6LinkLocalIP failed", ipsecIPv6Wildcard, cidr, spi)
 			}
-			spi, err := ipsec.UpsertIPsecEndpoint(linkAddr, ipsecIPv6Wildcard, cidr, ipsec.IPSecDirIn)
+			spi, err := ipsec.UpsertIPsecEndpoint(linkAddr, ipsecIPv6Wildcard, cidr, ipsec.IPSecDirIn, false)
```

Comment:
I'm not sure I understand the significance of passing `false` for this and subsequent changes in the rest of this file vs. earlier. What's the intent here?

Reply:
This will determine if the packet is routed back to the ingress interface or not. If true the packet will be routed directly from xfrm decrypt operation with being sent to program running on ingress interface. If false we will continue to run through ingress interface decrypted.

We could also set enableIPsec case true as well, but I didn't here and will to that in a follow up as its getting close to a feature at that point.

Reply:
I see, I think the piece of context I was missing is that this may only be true for subnet mode + endpoint routes (just like the commit message for that change implies).